src/main/java/com/adaptavist/tm4j/jenkins/extensions/configuration/Tm4jGlobalConfiguration.java

```diff
@@ -69,7 +69,7 @@ public boolean configure(StaplerRequest request, JSONObject formData) throws For
 
     private JiraInstance createAnInstance(JSONObject formJiraInstance) throws Exception {
         JiraInstance jiraInstance = new JiraInstance();
-        String serverAddres = formJiraInstance.getString("serverAddress");
+        String serverAddres = formJiraInstance.getString("validateServerAddress");
```

Comment:
I think this shouldn't have changed... the form field is still called `serverAddress` right?

Reply:
Sorry, IntelliJ refactoring :(